.github/workflows/build_and_test.yml

```diff
@@ -398,7 +402,7 @@ jobs:
       shell: bash
       run: |
         cd OpenDDS
-        tar cvf ../${{ github.job }}.tar setenv.sh install_setenv.sh
+        tar cvf ../${{ github.job }}.tar setenv.sh install_setenv.sh ACE_TAO
```

Comment:
This is a pretty significant change.  It's going to archive all of `ACE_TAO` and not just the files created by the build.

Reply:
I'm pretty sure this is was it doing this before with OCI TAO. When I changed it to use `--ace-github-latest` it was missing PerlACE because ACE_TAO is a git repository, so it was ignored. I guess I could limit it to archiving PerlACE though.

Reply:
Previously this one used `--doc-group` instead of `--ace-github-latest`.  So `git clean` would have seen all of the files as added.